src/Pages/ForgetPassword/ForgetPassword.tsx

```diff
@@ -1,10 +1,94 @@
-import React from 'react'
-import './ForgetPassword.module.scss'
+import {
+  Box,
+  Button,
+  FormControl,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import axios from "axios";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import baseUrl from "../../utils/Custom/Custom";
+import "./ForgetPassword.module.scss";
 
 const ForgetPassword = () => {
-  return <>
-    <div>ForgetPassword</div>
-  </>
-}
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  const onSubmit = (data) => {
+    axios
+      .post(`${baseUrl}/api/v0/portal/users/forgot-password`, data)
+      .then((res) => {
+        console.log("data", data);
```

Comment:
we need to remove logs before merging to the main branch to avoid revealing sensitive data.